test/v2/controllers/V2DeparturesControllerSpec.scala

```diff
@@ -272,29 +284,9 @@ class V2DeparturesControllerSpec
       Seq(HeaderNames.ACCEPT -> "application/vnd.hmrc.2.0+json", HeaderNames.CONTENT_TYPE -> "application/xml", HeaderNames.CONTENT_LENGTH -> "1000")
     )
 
-    "must return BadRequest when content length is not sent" in {
-      val departureHeaders = FakeHeaders(
-        Seq(HeaderNames.ACCEPT -> "application/vnd.hmrc.2.0+json", HeaderNames.CONTENT_TYPE -> "application/xml")
-      )
-      // We emulate no ContentType by sending in a stream directly, without going through Play's request builder
-      val request =
-        fakeRequestDepartures(method = "POST", body = Source.single(ByteString(CC015C.mkString, StandardCharsets.UTF_8)), headers = departureHeaders)
-      val result = app.injector.instanceOf[V2DeparturesController].submitDeclaration()(request)
-      status(result) mustBe BAD_REQUEST
-    }
-
-    "must return RequestEntityTooLarge when body size exceeds limit" in {
-      val departureHeaders = FakeHeaders(
-        Seq(HeaderNames.ACCEPT -> "application/vnd.hmrc.2.0+json", HeaderNames.CONTENT_TYPE -> "application/xml", HeaderNames.CONTENT_LENGTH -> "500001")
-      )
-      val request = fakeRequestDepartures(method = "POST", body = CC015C, headers = departureHeaders)
-      val result  = route(app, request).value
-      status(result) mustBe REQUEST_ENTITY_TOO_LARGE
-    }
```

Comment:
These tests have been removed, as the actions that these were testing are tested on the actions.